CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## 2.0.0 (July, 15, 2015)
+
+### features
+
+- Use API V2
+- Re-release the changes of 1.5.0 and 1.5.1 as 2.0.0
```

Comment:
This was done as you can see here https://github.com/Ink/filepicker-rails/blob/master/CHANGELOG.md